src/client/components/informationPanel/InformationCard.js

```diff
@@ -91,6 +91,7 @@ const StyledCard = styled(Card).attrs({ width: '100%', height: 'auto' })`
 const EthAddr = styled(Text).attrs({ size: 'small' })`
   color: ${theme.textTertiary};
   word-break: break-all;
+  font-size: 12px;
```

Comment:
Can we change the `attrs` to send the appropriate `size` prop to `<Text>` and avoid this override?

Reply:
Yes!